pkgs/servers/sql/materialize/default.nix

```diff
@@ -87,11 +87,13 @@ rustPlatform.buildRustPackage rec {
     install --mode=444 -D ./misc/dist/materialized.service $out/etc/systemd/system/materialized.service
   '';
 
+  doCheck = !stdenv.isDarwin;
```

Comment:
Please add a comment why this is necessary

Reply:
Okay, I've added a note. In general, I'm not sure if this considered standard practice, so please do feel free to reject the PR if so.